common/py/llamaindex/openai.py

```diff
@@ -1,22 +1,34 @@
+from agentuity import AgentRequest, AgentResponse, AgentContext
 from llama_index.core.agent.workflow import AgentWorkflow
 from llama_index.llms.openai import OpenAI
-from agentuity import AgentRequest, AgentResponse, AgentContext
-
-
-# Define a simple calculator tool
-def multiply(a: float, b: float) -> float:
```

Comment:
Weird black sheep of the bunch. I assume this was done to show off tools? Do we care? Presumably if they picked LlamaIndex they know how it works, and these are starter templates, not examples.